app/src/main/AndroidManifest.xml

```diff
@@ -12,6 +12,15 @@
     <uses-permission android:name="android.permission.FOREGROUND_SERVICE" />
     <uses-permission android:name="android.Manifest.permission.MEDIA_CONTENT_CONTROL" />
 
+    <uses-feature android:name="android.hardware.touchscreen"
+        android:required="false" />
+    <uses-feature
+        android:name="android.software.leanback"
+        android:required="false" />
+    <uses-feature
```

Comment:
But we do use camera for barcode scanning, right? What happens to that with this entry?

Reply:
Thinking more, I think it might be sufficient to hide that activity from the menu [if camera is not available](https://stackoverflow.com/a/17801627/5557184) on the system.

Reply:
https://developer.android.com/guide/topics/manifest/uses-feature-element#declared

Reply:
The hiding activity part works for me, will do that shortly.